src/Api.elm

```diff
@@ -9,14 +9,13 @@ import Route exposing (Route)
 placeCalApiUrl : String
 placeCalApiUrl =
     -- Production
-    "https://placecal.org/api/v1/graphql"
+    -- "https://placecal.org/api/v1/graphql"
 
+    -- Dev
+    -- "http://localhost:3000/api/v1/graphql"
 
-
--- Dev
---"http://lvh.me:3300/api/v1/graphql"
--- Staging
---"https://placecal-staging.org/api/v1/graphql"
+    -- Staging
+    "https://placecal-staging.org/api/v1/graphql"
```

Comment:
This is the dangerous one - so I'd remove it, and reinstate the dev one as an option for people.

My definition of danger is - can we accidentally use in production.

Reply:
I'm anyway going to make a backlog ticket, so happy for you to just checkout all these changes as they have no relation to this PR.

Reply:
I switched over to staging (as that is how it was initially) as the tests on the CI server were failing to connect to localhost:3000. 
I will have a deeper look at some kind of configuration system so we don't have to keep twiddling these values. Not sure how to fix the underlying problem of our CI build system being reliant on PC staging tho.

Reply:
It should be on production as per current head:
https://github.com/GenderedIntelligence/the-trans-dimension/blob/96f3be3ce54d9a47267ba1d785636e480897f70b/src/Api.elm

Reply:
This file should not have any changes in this PR.